tests/integration/pilot/locality/main_test.go

```diff
@@ -125,7 +131,7 @@ func init() {
 func TestMain(m *testing.M) {
 	framework.NewSuite("locality_prioritized_failover_loadbalancing", m).
 		Label(label.CustomSetup).
-		SetupOnEnv(environment.Kube, istio.Setup(&ist, setupConfig)).
```

Comment:
In surprised lint doesn't fail here for setupConfig not being used anymore

Reply:
It did after I fixed some other tests, weird, fixed it now.